data-imports/clean_utils.py

```diff
@@ -81,7 +82,7 @@ def add_boroid(table, column):
         """.format(table=lookup_table)
     if not column_exists(table, 'bbl'):
         sql = sql + """
-        ALTER TABLE {table} ADD COLUMN {columnid} bigint(13) NOT NULL DEFAULT 0;
+        ALTER TABLE {table} ADD COLUMN {columnid} bigint(13) NULL DEFAULT NULL;
```

Comment:
Why is this changed? I just changed this to "NOT NULL" to avoid breakage of some of the data-imports.